Cargo.toml

```diff
@@ -1,18 +1,13 @@
 [package]
 name = "hwid"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["S Stewart <iamtheworst@programmer.net>"]
-edition = "2018"
+edition = "2021"
+license-file = "LICENSE"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[dependencies]
+# https://stackoverflow.com/a/51821632/364875
+[target.'cfg(target_os = "windows")'.dependencies]
 winreg = "0.6"
 
-[lib]
-name = "hwid"
-path = "src/lib.rs"
-
-[[bin]]
```

Comment:
could add it back:
```toml
[[example]]
name = "get_hwid"
crate-type = ["bin"]
```

Reply:
Up to you. I think the simpler config is better, but I'm fine either way.

Reply:
Fair enough - won't force it in this PR at least